tests/mesolve/test_rouchon.py

```diff
@@ -24,11 +24,7 @@ def test_rho_save(self):
 
     def test_adjoint(self):
         options = dq.options.Rouchon1(dt=1e-3)
-        self._test_adjoint(options, grad_leaky_cavity_8, num_t_save=11)
-
-    def test_adjoint_expops(self):
-        options = dq.options.Rouchon1(dt=1e-3)
-        self._test_adjoint_expops(options, grad_leaky_cavity_8, num_t_save=11)
+        self._test_adjoint(options, grad_leaky_cavity_8, num_t_save=11, atol_exp=2e-1)
```

Comment:
See comment above.

Reply:
(So, in essence, it's strange to raise the atol to 0.2 for all tests here, when it's only needed for the one gradient that's very large and which features "only" a 2% error)